data/wanderer/wanderers.txt

```diff
@@ -767,7 +767,7 @@ event "wanderers: unfettered invasion starts"
 		fleet "Small Unfettered" 1800
 		fleet "Large Unfettered" 800
 	system "Ehma Ti"
-		remove fleet "Unfettered Tribute"
+		remove fleet "Unfettered Tribute (Wanderer)"
```

Comment:
Line 712 there is another one, you should change it as well if you want to rename the fleet
Or just dont rename it

Reply:
Good catch